src/lib/Components/Consignments/Screens/Welcome.tsx

```diff
@@ -1,114 +1,153 @@
 import { Button } from "../Components/FormElements"
-import { LargeHeadline } from "../Typography"
+import * as Typography from "../Typography"
 import Overview from "./Overview"
 
 import { Fonts } from "lib/data/fonts"
-import styled from "styled-components/native"
+import styled, { StyledFunction } from "styled-components/native"
+
+const View: StyledFunction<DeviceProps & ViewProperties> = styled.View
+const Text: StyledFunction<DeviceProps & TextProperties> = styled.Text
+
+interface DeviceProps {
+  isTiny: boolean
+  isPad: boolean
+  isPortrait: boolean
+}
 
 interface Props extends ViewProperties {
   navigator: NavigatorIOS
   route: Route
 }
 
-const isPad = Dimensions.get("window").width > 700
+interface State {
+  deviceProps?: DeviceProps
+}
 
-const VerticalLayout = styled.View`
+const VerticalLayout = View`
   flex: 1;
   flex-direction: column;
-  align-items: ${isPad ? "center" : "stretch"};
+  align-items: ${({ isPad }) => (isPad ? "center" : "stretch")};
 `
-const Listpad = styled.View`
-  align-items: center;
-  padding-top: 100;
-`
-const Listphone = styled.View`
-  margin-bottom: 100;
-  margin-left: 20;
-  margin-top: 50;
-  margin-right: 50;
+
+const List = View`
+  align-self: center;
+  width: ${({ isTiny }) => (isTiny ? 280 : 330)};
+  margin-top: 0;
+  margin-bottom: 0;
 `
 
-const HorizontalLayout = styled.View`
+const HorizontalLayout = View`
   flex-direction: row;
-  margin-left: ${isPad ? 0 : 20};
-  margin-top: ${isPad ? 40 : 10};
-  margin-bottom: ${isPad ? 0 : 10};
+  margin-left: 0;
+  margin-top: 0;
+  margin-bottom: 51;
 `
 
-const Title = styled.Text`
-  text-align: center;
-  font-size: ${isPad ? 20 : 16};
-  line-height: ${isPad ? 24 : 32};
-  width: ${isPad ? 760 : 300};
-  margin-top: ${isPad ? 110 : 50};
+const Title = Text`
+  font-size: 16;
+  line-height: 22;
+  margin-top: ${({ isPad, isPortrait }) => (isPad && isPortrait ? 230 : 58)};
   font-family: ${Fonts.AvantGardeRegular};
-  align-self: center;
   color: white;
+  margin-bottom: 63;
+  /* This text aligment doesn’t seem to work, so using letter-spacing instead. */
+  /* text-align: justify; */
+  letter-spacing: ${({ isTiny }) => (isTiny ? "-0.2" : "1.474")};
 `
 
-const SmallHeadline = styled(LargeHeadline)`
-  font-size: ${isPad ? 30 : 20};
+/* TODO: This should actually have a reduced line gap, but there doesn’t appear to be a way to do so. Maybe native? */
+const BaseHeadline: StyledFunction<DeviceProps & TextProperties> = styled(Typography.LargeHeadline)
+const SmallHeadline = BaseHeadline`
+  font-size: 20;
+  line-height: 22;
+  flex: 1;
+  padding: 0;
   color: white;
-  ${isPad ? "width: 540" : "max-width: 280"};
+  margin-right: 0;
+  margin-bottom: 0;
   text-align: left;
+  /* This is not 44, as per the design, because RN currently clips text regardless of overflow setting. In our case it
+     would clip the descender part of the font. */
+  height: 47;
+  /* height: 47; */
```

Comment:
Will this be uncommented later? Though it'd be a double height declaration?

Reply:
Oops, the commented one is supposed to reflect the design, which is 44pts.

Reply:
aaahh 😄  makes sense